install.sh

```diff
@@ -342,6 +342,10 @@ nvm_check_global_modules() {
   fi
 }
 
+nvm_install_manpage() {
+    ln -s "$PWD"/nvm.1 /usr/local/share/man/man1/nvm.1
```

Comment:
in the current working directory? or should this be inside `$NVM_DIR`?

Reply:
This is still from my previous hand-written attempt, and, along with the concern from @uberhacker earlier (much, much earlier), is subject to change.

Reply:
Same question tho, can i get a demo of what the `help2man` output would look like?